apps/site/lib/site_web/controllers/schedule/line_api.ex

```diff
@@ -51,17 +51,21 @@ defmodule SiteWeb.ScheduleController.LineApi do
   The line diagram polls this endpoint for its real-time data.
   """
   @spec realtime(Plug.Conn.t(), map()) :: Plug.Conn.t()
-  def realtime(conn, %{"id" => route_id, "direction_id" => direction_id}) do
-    cache_key = {route_id, direction_id, conn.assigns.date}
+  def realtime(conn, %{"id" => route_id, "direction_id" => direction_id} = query_params) do
+    if query_params["v"] == "2" do
```

Comment:
We should leave a comment here as to why we're doing this, so future generations (or just us, later) will feel confident in removing and/or augmenting it.